gluetool/glue.py

```diff
@@ -1,14 +1,18 @@
 # pylint: disable=too-many-lines
 
 import argparse
-import ConfigParser
+try:
+    import configparser
+except:
+    from six.moves import configparser
```

Comment:
Just `from six.moves import configparser`, no need to bother with import error, `six` will take care of it.